### DISCUSSION TITLE
Is there a React Native module for SIWS?

### DISCUSSION THREAD

**Discussion by metasal1** [phantom/docs#98] [Help & Support] [answered]
### Summary

We are building an app for Android and iPhone, we need to know if there is an SIWS for React Native

### Steps to Reproduce

N/A

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hey @metasal1,

Currently SIWS is only supported on Phantom Extension >= 23.11.0 and on Phantom Mobile in the in-app browser >= 23.13.0, which is coming very soon. SIWS deeplinking is not currently supported. Additionally there is no react native module available at this time.

Please keep an eye on our Changelog available in our [Announcements](https://github.com/orgs/phantom/discussions/categories/announcements) section for future updates and options.
